posts/inside-rust/2020-03-12-twir-new-lead.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: "Vikrant (nasa42) is retiring from This Week in Rust. TWiR is looking for a new maintainer."
+author: <TBD>
+team: the community team <https://www.rust-lang.org/governance/teams/community>
+---
+
+Vikrant is retiring from This Week in Rust. He joined This Week in Rust in June 2015 with issue 84 and has been part of Rust Community team since February 2018.
```

Comment:
```suggestion
Vikrant Chaudhary is retiring from [This Week in Rust][twir]. He joined This Week in Rust in June 2015 with issue 84 and has been part of Rust Community team since February 2018. We'd like to thank Vikrant for his stewardship of TWiR these past five years, and making TWiR one of the community's favourite newsletters. We wish him all the best in his future projects.

[twir]: https://this-week-in-rust.org
```